### PR TITLE
Docs.

### PR DESCRIPTION
**Pull request by montera82**

### CODE REVIEW

File: packages/snap/README.md [hunk 1]
```diff
@@ -1,8 +1,60 @@
...
+```
+
+## Usage
+
+To interact with this Snap, you will need to install [MetaMask Flask](https://metamask.io/flask/), a canary distribution for developers that provides access to upcoming features.
+
+Then use Snap ID to connect to this Snap:
+
+```javascript
+const snapId = 'npm:@celo/gas-snap';
```
Comment (montera82) [inline packages/snap/README.md:20]: this is a place holder, as snap has not yet been publlished. waiting for npm access to fix that.

Comment (aaronmgdr) [reply packages/snap/README.md:20]: best would be to talk with @dckesler  or @pputman-clabs about getting automated publishing set up

Comment (montera82) [reply packages/snap/README.md:20]: yea thats a good idea, for now we want to deploy manually via cli

File: packages/snap/README.md [hunk 2]
```diff
@@ -1,8 +1,67 @@
...
-## Notes
+[![Open Source Love svg1](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/ellerbrock/open-source-badges/)
 
-- Babel is used for transpiling TypeScript to JavaScript, so when building with the CLI,
-  `transpilationMode` must be set to `localOnly` (default) or `localAndDeps`.
+## Install
+
+```
+yarn add @celo/gas-snap
+```
```
Comment (aaronmgdr) [inline packages/snap/README.md:11]: do you actually need to install this like this just to use it?

File: packages/snap/README.md [hunk 3]
```diff
@@ -1,8 +1,61 @@
...
+
+Once connected, you can call the RPC method exposed in this snap as follows:
+
+```javascript
+
+ import { CeloTransactionRequest } from '@celo-tools/celo-ethers-wrapper';
+
+ const tx: CeloTransactionRequest = {
+        to: <address>,
+        value: Number(ethers.utils.parseUnits("1", "wei"))
```
Comment (alecps) [inline packages/snap/README.md:40]:
```suggestion
        value: ethers.utils.parseUnits("1", "wei")
```

**Review by alecps — APPROVED** [review]

File: packages/snap/README.md (hunk 3 above)
Comment (alecps) [reply packages/snap/README.md:40]: You don't need to wrap this in a number here, the type is already correct

**Pull request merged**